### PR TITLE
feat(slack-notifications): [nan-1148] slack notification UI polish

### PR DESCRIPTION
**Pull request by khaliqgant**
## Describe your changes
Update the slack notification to link to the connection and also integration. Adjust the admin notification to include the customer name and uuid in the footer

## Issue ticket number and link
NAN-1148

## Checklist before requesting a review (skip if just adding/editing APIs & templates)
- [ ] I added tests, otherwise the reason is: 
- [ ] I added observability, otherwise the reason is:
- [ ] I added analytics, otherwise the reason is:

### CODE REVIEW

File: packages/server/nango-integrations/.nango/schema.json [hunk 1]
```diff
@@ -0,0 +1,136 @@
+{
```
Comment (bodinsamuel) [inline packages/server/nango-integrations/.nango/schema.json]: No need to commit the files in `.nango` folder

**Review by bodinsamuel — APPROVED** [review]
Looks about right, not tested locally 👍🏻 
How is this deployed to prod? it's manual right?

File: packages/server/nango-integrations/.nango/schema.json (hunk 1 above)
Comment (khaliqgant) [reply packages/server/nango-integrations/.nango/schema.json]: Yeah, removed.

**Comment by khaliqgant** [comment]
> Looks about right, not tested locally 👍🏻 How is this deployed to prod? it's manual right?

Indeed. From the project root `npm run nango deploy prod`

**Review by TBonnin — APPROVED** [review]

**Pull request merged**